### PR TITLE
Update urls for rhel 7.7 and 7.8

### LINKED ISSUE #734: Image refresh for rhel-7-8
Image refresh for rhel-7-8
 * [ ] FAIL: image-refresh rhel-7-8

### PR DESCRIPTION
**Pull request by marusak**
* [x] image-refresh rhel-7-7
 * [x] image-refresh rhel-7-8

Fixes #734

### CODE REVIEW

**Comment by cockpituous** [comment]
image-refresh in progress on 3-cockpit-10.
Log: https://logs.cockpit-project.org/logs/image-refresh-758-20200416-085151/

File: images/scripts/rhel-7-7.bootstrap [hunk 1]
```diff
@@ -3,4 +3,4 @@
 set -ex
 
 BASE=$(dirname $0)
-$BASE/virt-install-fedora "$1" x86_64 "http://download.devel.redhat.com/released/RHEL-7/7.7/Server/x86_64/os/"
+$BASE/virt-install-fedora "$1" x86_64 "http://download.devel.redhat.com/rhel-7/rel-eng/latest-RHEL-7.7/compose/Server/x86_64/os/"
```
Comment (martinpitt) [inline images/scripts/rhel-7-7.bootstrap:6]: The same change needs to happen in rhel.setup.

**Comment by cockpituous** [comment]
image-refresh rhel-7-7 done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-7-7-20200416-091515

**Comment by cockpituous** [comment]
image-refresh in progress on 4-cockpit-9.
Log: https://logs.cockpit-project.org/logs/image-refresh-758-20200416-091515/

**Comment by cockpituous** [comment]
Task failed: https://logs.cockpit-project.org/logs/image-refresh-758-20200416-091515/

**Comment by cockpituous** [comment]
image-refresh in progress on 4-ci-srv-04.
Log: https://logs.cockpit-project.org/logs/image-refresh-758-20200416-094545/

**Comment by cockpituous** [comment]
image-refresh rhel-7-7 done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-7-7-20200416-101010

**Comment by cockpituous** [comment]
image-refresh in progress on 1-cockpit-11.
Log: https://logs.cockpit-project.org/logs/image-refresh-758-20200416-101010/

**Comment by cockpituous** [comment]
image-refresh rhel-7-8 done: https://github.com/cockpituous/bots/commits/image-refresh-rhel-7-8-20200416-103636

**Comment by marusak** [comment]
The usual, [sos report](https://logs.cockpit-project.org/logs/pull-758-20200416-145120-528c21bd-rhel-atomic-cockpit-project-cockpit-rhel-7.9/log.html) on atomic, [openshift](https://logs.cockpit-project.org/logs/pull-758-20200416-145114-528c21bd-rhel-7-8-cockpit-project-cockpit-rhel-7.8/log.html) on 7.8, [vmware](https://logs.cockpit-project.org/logs/pull-758-20200416-145124-528c21bd-rhel-7-8-vmware-weldr-lorax-rhel7-extras/log.html) and [openstack](https://logs.cockpit-project.org/logs/pull-758-20200416-145116-528c21bd-rhel-7-8-openstack-weldr-lorax-rhel7-extras/log.html) on lorax. I'll spin them once more, but I guess it is gonna be the same.

**Pull request merged**